src/Microsoft.CodeAnalysis.Testing/Microsoft.CodeAnalysis.Analyzer.Testing/AnalyzerTest`1.cs

```diff
@@ -822,6 +823,13 @@ bool IsCompilerDiagnosticIncluded(Diagnostic diagnostic)
             }
         }
 
+        /// <summary>
+        /// Extension point to allow inheriting classes to include diagnostics which would otherwise be filtered. The default implementation returns false
+        /// </summary>
+        /// <param name="diagnostic">the diagnostic which is subject to filter processes.</param>
+        /// <returns>return true to exclude a diagnostic, false to leave it up to internal logic.</returns>
+        protected internal virtual bool IsDiagnosticHandled(Diagnostic diagnostic) => false;
```

Comment:
❔ Can this be just `protected`?

Reply:
My bad, makes sense. Otherwise it's a pretty bad extension point.